Ontology/Willow/Component/Asset/Fan.json

```diff
@@ -71,6 +71,16 @@
       "schema": "PowerUnit",
       "writable": true
     },
+    {
+      "@type": ["AngularVelocity", "Property"],
+      "displayName": {
+        "en": "maximum rotation speed"
+      },
+      "name": "maxRotationSpeed",
+      "schema": "double",
+      "unit": "revolutionPerMinute",
```

Comment:
For fan max speed, we're usually looking for the max output frequency of the VFD, i.e. 60Hz, 90Hz, etc. I think we should either add that or replace the angular velocity with it

Reply:
Max and Actual RPMs were from an AHU submittal. That particular sheet didn't include VFD output frequency but easy to include it too.

Reply:
Very good, let's keep them all